packages/apps/job-launcher/server/src/modules/payment/payment.service.ts

```diff
@@ -78,21 +74,16 @@ export class PaymentService {
   }
 
   public async createFiatPayment(
-    customerId: string,
+    user: UserEntity,
```

Comment:
Where are you using this `user` object here?

Reply:
Let’s keep the user inside create payment intent I will use it for the next issue related with client secret